hw/ip/otbn/dv/verilator/otbn_top_sim.cc

```diff
@@ -12,6 +12,8 @@
 #include "verilator_sim_ctrl.h"
 
 extern "C" {
+extern unsigned int otbn_base_call_stack_get_size();
+extern unsigned int otbn_base_call_stack_get_element(int index);
```

Comment:
We've got two subtly different ways to get at the design state (RF and call stack). I didn't want to address this in this PR but I think we should either ditch the register/call stack dump at the end of sim or switch to using the snoopers used for the model cross-check.